### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by tarasavage**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_name, file, file_copy = command.split(" ")
```
Comment (Grayder0152) [inline app/main.py:2]: You should check if `command` length == 3, before it

**Review by Grayder0152 — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 2]
```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split(" ")
+    if len(parts) == 3:
+        command_name, file, file_copy = parts
```
Comment (ArsenPidhoretskyi) [inline app/main.py:4]: file, file_copy not clear name
first of all it is source_file_path, destination_file_path

File: app/main.py [hunk 3]
```diff
@@ -1 +1,8 @@
...
+        if file != file_copy and command_name == "cd":
+            with (open(file, "r") as file_in,
+                  open(file_copy, "w") as file_out):
```
Comment (ArsenPidhoretskyi) [inline app/main.py:7]:
```suggestion
            with (
               open(file, "r") as file_in,
               open(file_copy, "w") as file_out
            ):
```

Comment (ArsenPidhoretskyi) [reply app/main.py:7]: rename file_in, file_out to source_file, destination_file

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

**Review by vasylhnatiuk — APPROVED** [review]
Good job!

**Pull request closed without merging**